packages/website/pages/tokens.js

```diff
@@ -203,6 +203,14 @@ export default function Tokens({ isLoggedIn }) {
                 </div>
               </div>
             </When>
+            <When condition={tokens.length === 1}>
+              <div className="flex flex-col items-center">
+                <p className="font-black mt-10">
+                  Want to test the token quickly? Upload a file in 
+                  {" "}<a className="underline" href={`https://web3storage-tokens-srv.rafaelramalho1.repl.co/?token=${tokens[0].secret}`} target="_blank" rel="noreferrer">repl.it</a>
```

Comment:
We need to set up a web3.stroage account for repl.co rather than relying on `rafaelramalho1`

Reply:
Done :)